components/automate-ui/src/app/pages/data-feed-create/data-feed-create.component.scss

```diff
@@ -93,7 +93,7 @@ main {
     }
 
     .input-margin {
-      display: flex;
+      display: inline-block;
```

Comment:
```suggestion
      display: inline-block;
      vertical-align:top;
```
Please align vertical align as well because the elements shift to baseline in error state. 
<img width="649" alt="Screenshot 2021-09-16 at 8 12 40 PM" src="https://user-images.githubusercontent.com/59958706/133632975-3c3ddbd2-ca1d-4a9e-a88e-426db175bdcd.png">